src/graph/graphcontext_type.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Apache License, Version 2.0,
+ * modified with the Commons Clause restriction.
+ */
+
+#include "graphcontext.h"
+#include "graphcontext_type.h"
+#include "../index/index_type.h"
+
+/* Declaration of the type for redis registration. */
+RedisModuleType *GraphContextRedisModuleType;
+
+void GraphContextType_RdbSave(RedisModuleIO *rdb, void *value) {
+  GraphContext *gc = value;
+  RedisModule_SaveStringBuffer(rdb, gc->graph_name, strlen(gc->graph_name) + 1);
+
+  RedisModule_SaveUnsigned(rdb, gc->node_cap);
```

Comment:
We need to decide on exactly what is referenced from the context object and what is owned by it,
Let's try to discus this ASAP